agent/eni/watcher/watcher_linux.go

```diff
@@ -57,6 +88,17 @@ type UdevWatcher struct {
 	primaryMAC           string
 }
 
+// unmanagedENIError is used to indicate that the agent found an ENI, but the agent isn't
+// aware if this ENI is being managed by ECS
+type unmanagedENIError struct {
+	mac string
+}
+
+// Error returns the error string for the unmanagedENIError type
+func (err *unmanagedENIError) Error() string {
+	return fmt.Sprintf("udev watcher send ENI state change: eni not managed by ecs: %s", err.mac)
```

Comment:
it's not clear to me what we mean by `udev watcher send ENI state change`. the second part of the error message is fine though.